ui/src/components/config_section/ActivityConfigSection.js

```diff
@@ -1,10 +1,10 @@
+import "components/config_section/ActivityConfigSection.scss";
```

Comment:
Ooh I think this can still be a relative import as before. `"./ActivityConfigSection.scss"` can be thought of as being local to this component. Do you see any benefit to having the absolute import here?

I have the same comment about other similar changes in this PR. Generally, my rule of thumb is - avoid `../` in imports, at all costs. But, `./` is ok if the imported component can be thought of as a child of the current component or something that goes hand in hand like the style sheets - they will likely live together forever and be refactored together. And, I'd actually argue that the new change makes the import harder to read and picture the relationship to the component.

From a quick search, I didn't find any "best practices" around this online - whether we should adopt absolute imports everywhere or if relative imports are better suited for some scenarios. What are your thoughts?

CC: @leonlnj

Reply:
no opinions, as long as we agree and follow 😆

Reply:
Okay then, I'll revert `./` changes!